op-chain-ops/genesis/layer_one.go

```diff
@@ -91,3 +97,23 @@ func BuildL1DeveloperGenesis(config *DeployConfig, dump *gstate.Dump, l1Deployme
 
 	return memDB.Genesis(), nil
 }
+
+// FundDevAccounts will fund each of the development accounts.
+func FundDevAccounts(db vm.StateDB) {
+	for _, account := range DevAccounts {
+		if !db.Exist(account) {
+			db.CreateAccount(account)
+		}
+		db.AddBalance(account, uint256.MustFromBig(devBalance))
+	}
+}
+
+// SetPrecompileBalances will set a single wei at each precompile address.
+// This is an optimization to make calling them cheaper.
+func SetPrecompileBalances(db vm.StateDB) {
+	for i := 0; i < PrecompileCount; i++ {
+		addr := common.BytesToAddress([]byte{byte(i)})
+		db.CreateAccount(addr)
+		db.AddBalance(addr, uint256.NewInt(1))
+	}
+}
```

Comment:
Can we just refactor the graceful account creation here into another function and then dogfood?

```suggestion
// CreateAccountNotExists creates the account in the `vm.StateDB` if it doesn't exist.
func CreateAccountNotExists(db vm.StateDB, account common.Address) {
	if !db.Exist(account) {
		db.CreateAccount(account)
	}
}

// FundDevAccounts will fund each of the development accounts.
func FundDevAccounts(db vm.StateDB) {
	for _, account := range DevAccounts {
		CreateAccountNotExists(db, account)
		db.AddBalance(account, uint256.MustFromBig(devBalance))
	}
}

// SetPrecompileBalances will set a single wei at each precompile address.
// This is an optimization to make calling them cheaper.
func SetPrecompileBalances(db vm.StateDB) {
	for i := 0; i < PrecompileCount; i++ {
		addr := common.BytesToAddress([]byte{byte(i)})
		CreateAccountNotExists(db, addr)
		db.AddBalance(addr, uint256.NewInt(1))
	}
}
```